LICENSE.enterprise

```diff
@@ -0,0 +1,5 @@
+Copyright 2022 Coder Technologies, Inc. All Rights Reserved.
+
+License TBD
+
+
```

Comment:
Assuming all the TBD-licensed code lives in `enterprise`, should this live in there as well?

Reply:
I'm following the example in https://github.com/sourcegraph/sourcegraph for reference